File: build_brief_orbitkey.js
const pptxgen = require("pptxgenjs");

// Brand colours (Rule 3), no hash prefix
const BLACK = "000000";
const TEAL = "01A296";
const AMBER_BRIGHT = "F8C806";
const AMBER_DEEP = "F6A102";
const GOLD = "E3A712";
const WHITE = "FFFFFF";
const OFF_WHITE = "E6E5DE";

const HEADER_SERIF = "Cambria";
const BODY_SANS = "Calibri";

const DATE_STAMP = "31 Jul 2026";
const COMPANY = "Orbitkey";

function newDeck() {
  const pres = new pptxgen();
  pres.layout = "LAYOUT_WIDE"; // 13.333 x 7.5
  return pres;
}

function addChrome(slide, eyebrow, rightLabel) {
  // page background
  slide.background = { color: WHITE };
  // left accent stripe
  slide.addShape("rect", { x: 0, y: 0, w: 0.1, h: 7.5, fill: { color: AMBER_BRIGHT }, line: { type: "none" } });
  // header band
  slide.addShape("rect", { x: 0, y: 0, w: 13.333, h: 1.0, fill: { color: BLACK }, line: { type: "none" } });
  slide.addText(eyebrow, {
    x: 0.3, y: 0.32, w: 7.5, h: 0.4, fontFace: BODY_SANS, fontSize: 12, bold: true,
    color: OFF_WHITE, align: "left", margin: 0,
  });
  slide.addText(rightLabel, {
    x: 9.5, y: 0.32, w: 3.5, h: 0.4, fontFace: BODY_SANS, fontSize: 12, bold: true,
    color: TEAL, align: "right", margin: 0,
  });
  // footer line
  slide.addText("Prepared by Nitesh Roopa CA, Managing Partner and Founder, ProfitPulse, Profit-Pulse.com.au", {
    x: 0.3, y: 7.05, w: 9.0, h: 0.3, fontFace: BODY_SANS, fontSize: 9, color: BLACK, align: "left", margin: 0,
  });
  slide.addText(DATE_STAMP, {
    x: 10.0, y: 7.05, w: 3.0, h: 0.3, fontFace: BODY_SANS, fontSize: 9, color: BLACK, align: "right", margin: 0,
  });
}

const pres = newDeck();

// ─────────────────────────────────────────────────────────────────────────
// SLIDE 1: COMMERCIAL INTELLIGENCE BRIEF
// ─────────────────────────────────────────────────────────────────────────
const s1 = pres.addSlide();
addChrome(s1, "COMMERCIAL INTELLIGENCE BRIEF", "PROFITPULSE");

s1.addText(COMPANY, {
  x: 0.35, y: 1.12, w: 9.0, h: 0.75, fontFace: HEADER_SERIF, fontSize: 42, bold: true,
  color: BLACK, align: "left", margin: 0,
});
s1.addText("Design led everyday carry accessories, Richmond, Melbourne VIC", {
  x: 0.35, y: 1.85, w: 10.5, h: 0.35, fontFace: BODY_SANS, fontSize: 13, italic: true,
  color: BLACK, align: "left", margin: 0,
});

// Stat cards row: 5 cards, 2.3" wide, 0.15" gap, top y 2.4, height 1.6
const cardY = 2.4, cardW = 2.3, cardH = 1.6, cardGap = 0.15, cardStartX = 0.35;
const cards = [
  { num: "$24.6M", lbl: "Annual revenue", src: "SmartCompany Smart50 2025" },
  { num: "20%", lbl: "Average 3 year growth", src: "SmartCompany Smart50 2025" },
  { num: "44", lbl: "People on the team", src: "SmartCompany Smart50 2025" },
  { num: "83.7", lbl: "B Corp impact score", src: "B Lab Global directory" },
  { num: "$8M+", lbl: "Raised via Kickstarter", src: "SmartCompany, 11 campaigns" },
];
cards.forEach((c, i) => {
  const x = cardStartX + i * (cardW + cardGap);
  s1.addShape("rect", { x, y: cardY, w: cardW, h: cardH, fill: { color: BLACK }, line: { type: "none" } });
  s1.addShape("rect", { x, y: cardY, w: cardW, h: 0.06, fill: { color: TEAL }, line: { type: "none" } });
  s1.addText(c.num, {
    x: x + 0.15, y: cardY + 0.16, w: cardW - 0.3, h: 0.5, fontFace: HEADER_SERIF, fontSize: 27, bold: true,
    color: AMBER_BRIGHT, align: "left", margin: 0,
  });
  s1.addText(c.lbl, {
    x: x + 0.15, y: cardY + 0.68, w: cardW - 0.3, h: 0.5, fontFace: BODY_SANS, fontSize: 11,
    color: OFF_WHITE, align: "left", margin: 0, valign: "top",
  });
  s1.addText(c.src, {
    x: x + 0.15, y: cardY + cardH - 0.32, w: cardW - 0.3, h: 0.26, fontFace: BODY_SANS, fontSize: 7.5,
    color: TEAL, align: "left", margin: 0,
  });
});

// Key Commercial Signals
s1.addText("KEY COMMERCIAL SIGNALS", {
  x: 0.35, y: 4.22, w: 6.0, h: 0.3, fontFace: BODY_SANS, fontSize: 12, bold: true,
  color: TEAL, align: "left", margin: 0,
});
const signals = [
  "B Corp Certified, impact score 83.7 vs global median 50.9 (B Lab Global).",
  "2025 Australian Export Award winner, Creative Industries (exportawards.gov.au).",
  "New Richmond HQ built beside the Kensington design studio (company sources).",
  "Latest Kickstarter campaign raised over $393,500, no external investors (SmartCompany).",
  "South Korea flagged as a significant emerging export market (exportawards.gov.au).",
  "Smart50 2025 ranking, independent confirmation of continued fast growth (SmartCompany).",
];
s1.addText(
  signals.map((t, i) => ({ text: t, options: { bullet: { code: "2022" }, breakLine: i < signals.length - 1, color: BLACK, fontSize: 11.5, fontFace: BODY_SANS, paraSpaceAfter: 6 } })),
  { x: 0.35, y: 4.56, w: 12.4, h: 2.3, margin: 0, valign: "top" }
);

// ─────────────────────────────────────────────────────────────────────────
// SLIDE 2: THE OPPORTUNITY
// ─────────────────────────────────────────────────────────────────────────
const s2 = pres.addSlide();
addChrome(s2, "THE OPPORTUNITY", "PROFITPULSE");

s2.addText("Orbitkey. Three commercial observations from ProfitPulse.", {
  x: 0.35, y: 1.14, w: 12.4, h: 0.4, fontFace: HEADER_SERIF, fontSize: 19, bold: true,
  color: BLACK, align: "left", margin: 0,
});

const colY = 1.7, colH = 4.6, colW = 4.1, colGap = 0.15, colStartX = 0.35;
const observations = [
  {
    index: "01", header: "Stock now funds Orbitkey's growth", fill: TEAL, textColor: BLACK,
    para: "Twenty percent average growth with zero external capital means inventory must be bought before wholesale partners or Kickstarter backers pay Orbitkey back. With around 3,000 retail partners plus direct ecommerce, cash timing matters as much as sales. A working capital review shows what releasing cash could fund next.",
  },
  {
    index: "02", header: "The channel mix is getting harder to read", fill: BLACK, textColor: OFF_WHITE,
    para: "Orbitkey sells through direct ecommerce, around 3,000 wholesale partners, and periodic Kickstarter launches, three channels with different margins and cost to serve. As the wholesale network keeps growing internationally, not every channel is contributing equally once freight and 3PL terms are allocated properly.",
  },
  {
    index: "03", header: "Recognition is outrunning reporting", fill: GOLD, textColor: BLACK,
    para: "A B Corp certification, a national Export Award, and a Smart50 ranking in one year put Orbitkey in front of partners and future capital sources expecting board grade numbers on request. Staying self funded is a real strength, but it still benefits from the same monthly rhythm investor backed rivals must run.",
  },
];
observations.forEach((o, i) => {
  const x = colStartX + i * (colW + colGap);
  s2.addShape("rect", { x, y: colY, w: colW, h: colH, fill: { color: o.fill }, line: { type: "none" } });
  s2.addText(o.index, {
    x: x + 0.25, y: colY + 0.2, w: colW - 0.5, h: 0.55, fontFace: HEADER_SERIF, fontSize: 30, bold: true,
    color: o.textColor, align: "left", margin: 0,
  });
  s2.addText(o.header, {
    x: x + 0.25, y: colY + 0.85, w: colW - 0.5, h: 0.75, fontFace: HEADER_SERIF, fontSize: 15, bold: true,
    color: o.textColor, align: "left", margin: 0, valign: "top",
  });
  s2.addText(o.para, {
    x: x + 0.25, y: colY + 1.65, w: colW - 0.5, h: colH - 1.85, fontFace: BODY_SANS, fontSize: 10.5,
    color: o.textColor, align: "left", margin: 0, valign: "top", lineSpacingMultiple: 1.15,
  });
});

s2.addText(
  "These observations are offered in good faith. Orbitkey has built something impressive without taking a dollar of outside capital. The question is simply whether the financial architecture keeps pace with the ambition.",
  { x: 0.35, y: colY + colH + 0.12, w: 12.4, h: 0.5, fontFace: BODY_SANS, fontSize: 10.5, italic: true, color: BLACK, align: "left", margin: 0 }
);

// ─────────────────────────────────────────────────────────────────────────
// SLIDE 3: THE RECOMMENDATION
// ─────────────────────────────────────────────────────────────────────────
const s3 = pres.addSlide();
addChrome(s3, "THE RECOMMENDATION", "PROFITPULSE");

const leftX = 0.35, leftW = 7.65;
const rightX = 8.35, rightW = 4.6;

s3.addText("Working Capital Unlock", {
  x: leftX, y: 1.15, w: leftW, h: 0.5, fontFace: HEADER_SERIF, fontSize: 26, bold: true,
  color: BLACK, align: "left", margin: 0,
});
s3.addText("$6,000 one off", {
  x: leftX, y: 1.65, w: leftW, h: 0.4, fontFace: BODY_SANS, fontSize: 18, bold: true,
  color: AMBER_DEEP, align: "left", margin: 0,
});
s3.addText(
  "A four week project mapping cash trapped in stock, debtors, and supplier terms across five global warehouses, with a prioritised, quantified action list to release it.",
  { x: leftX, y: 2.12, w: leftW, h: 0.55, fontFace: BODY_SANS, fontSize: 12, color: BLACK, align: "left", margin: 0, valign: "top" }
);

// Step one block
s3.addShape("rect", { x: leftX, y: 2.85, w: leftW, h: 1.35, fill: { color: OFF_WHITE }, line: { type: "none" } });
s3.addText("Step one, answer a few quick questions", {
  x: leftX + 0.2, y: 2.98, w: leftW - 0.4, h: 0.35, fontFace: BODY_SANS, fontSize: 13, bold: true,
  color: BLACK, align: "left", margin: 0,
});
s3.addText("See the solutions matched to your size and industry.", {
  x: leftX + 0.2, y: 3.32, w: leftW - 0.4, h: 0.32, fontFace: BODY_SANS, fontSize: 11.5,
  color: BLACK, align: "left", margin: 0,
});
s3.addText("profit-pulse.com.au/services/find-your-fit", {
  x: leftX + 0.2, y: 3.66, w: leftW - 0.4, h: 0.4, fontFace: BODY_SANS, fontSize: 13, bold: true,
  color: TEAL, align: "left", margin: 0,
  hyperlink: { url: "https://profit-pulse.com.au/services/find-your-fit/" },
});

// Direct CTA
s3.addShape("roundRect", {
  x: leftX, y: 4.4, w: leftW, h: 0.55, rectRadius: 0.1,
  fill: { color: AMBER_DEEP }, line: { type: "none" },
});
s3.addText("Purchase the suggested product now to get started", {
  x: leftX, y: 4.4, w: leftW, h: 0.55, fontFace: BODY_SANS, fontSize: 13, bold: true,
  color: BLACK, align: "center", valign: "middle", margin: 0, wrap: false,
  hyperlink: { url: "https://buy.stripe.com/00w4gA7OUfGM5TadMH3ks0z" },
});

// Booking
s3.addText("Prefer a conversation first?", {
  x: leftX, y: 5.2, w: leftW, h: 0.32, fontFace: BODY_SANS, fontSize: 12, color: BLACK, align: "left", margin: 0,
});
s3.addText("Book a complimentary discovery call", {
  x: leftX, y: 5.52, w: leftW, h: 0.35, fontFace: BODY_SANS, fontSize: 12.5, bold: true,
  color: TEAL, align: "left", margin: 0,
  hyperlink: { url: "https://bookings.cloud.microsoft/book/ProfitPulse1@profit-pulse.com.au/?ismsaljsauthenabled=true" },
});

// Right column: credibility panel
s3.addShape("rect", { x: rightX, y: 1.15, w: rightW, h: 5.6, fill: { color: BLACK }, line: { type: "none" } });
s3.addText("NITESH ROOPA", {
  x: rightX + 0.25, y: 1.32, w: rightW - 0.5, h: 0.42, fontFace: HEADER_SERIF, fontSize: 17, bold: true,
  color: AMBER_BRIGHT, align: "left", margin: 0,
});
s3.addText("CA, Managing Partner, ProfitPulse", {
  x: rightX + 0.25, y: 1.75, w: rightW - 0.5, h: 0.35, fontFace: BODY_SANS, fontSize: 12,
  color: WHITE, align: "left", margin: 0,
});

const credLines = [
  "16 years across 4 countries",
  "52 deals executed and managed",
  "Largest single deal USD 1.3 billion, Cahora Bassa",
  "Total GRBT project value over AUD 10 billion",
];
s3.addText(
  credLines.map((t, i) => ({ text: t, options: { bullet: { code: "2022" }, breakLine: i < credLines.length - 1, color: OFF_WHITE, fontSize: 10.5, fontFace: BODY_SANS, paraSpaceAfter: 8 } })),
  { x: rightX + 0.25, y: 2.25, w: rightW - 0.5, h: 1.7, margin: 0, valign: "top" }
);

s3.addShape("line", { x: rightX + 0.25, y: 4.05, w: rightW - 0.5, h: 0, line: { color: TEAL, width: 1 } });

const contactLines = [
  "Profit-Pulse.com.au",
  "Nitesh@Profit-Pulse.com.au",
  "+61 411 876 267",
  "linkedin.com/in/nitesh-roopa-77594163",
];
s3.addText(
  contactLines.map((t, i) => ({ text: t, options: { breakLine: i < contactLines.length - 1, color: OFF_WHITE, fontSize: 10.5, fontFace: BODY_SANS, paraSpaceAfter: 6 } })),
  { x: rightX + 0.25, y: 4.25, w: rightW - 0.5, h: 1.4, margin: 0, valign: "top" }
);

pres.writeFile({ fileName: "/home/user/Daily-Outreach/Out-reach efforts/Brief_Orbitkey_31Jul2026.pptx" })
  .then(() => console.log("PPTX written"))
  .catch((e) => { console.error(e); process.exit(1); });

File: build_brief_orbitkey.py
"""
ProfitPulse Brief Builder
Target: Orbitkey | Date: 05 Aug 2026
Three slide prospect facing deck. House style per Section 6.0 and 6.0A. Brand colours only. Zero dashes.
"""

from pptx import Presentation
from pptx.util import Inches, Pt
from pptx.dml.color import RGBColor
from pptx.enum.text import PP_ALIGN, MSO_ANCHOR
from pptx.enum.shapes import MSO_SHAPE

BLACK      = RGBColor(0x00, 0x00, 0x00)
TEAL       = RGBColor(0x01, 0xA2, 0x96)
AMBER_B    = RGBColor(0xF8, 0xC8, 0x06)
AMBER_D    = RGBColor(0xF6, 0xA1, 0x02)
GOLD       = RGBColor(0xE3, 0xA7, 0x12)
WHITE      = RGBColor(0xFF, 0xFF, 0xFF)
OFF_WHITE  = RGBColor(0xE6, 0xE5, 0xDE)

W = Inches(13.333)
H = Inches(7.5)
DATE_STR = "05 Aug 2026"
COMPANY = "Orbitkey"
SLUG = "orbitkey"

prs = Presentation()
prs.slide_width = W
prs.slide_height = H
blank = prs.slide_layouts[6]


def bg(slide, colour):
    f = slide.background.fill
    f.solid()
    f.fore_color.rgb = colour


def rect(slide, x, y, w, h, fill, line=None):
    shp = slide.shapes.add_shape(MSO_SHAPE.RECTANGLE, x, y, w, h)
    shp.fill.solid()
    shp.fill.fore_color.rgb = fill
    if line:
        shp.line.color.rgb = line
        shp.line.width = Pt(1)
    else:
        shp.line.fill.background()
    shp.shadow.inherit = False
    return shp


def text(slide, s, x, y, w, h, size=12, bold=False, colour=BLACK,
         align=PP_ALIGN.LEFT, font="Calibri", anchor=None, italic=False,
         line_spacing=None, shrink=True):
    box = slide.shapes.add_textbox(x, y, w, h)
    tf = box.text_frame
    tf.word_wrap = True
    if shrink:
        from pptx.enum.text import MSO_AUTO_SIZE
        tf.auto_size = MSO_AUTO_SIZE.NONE
    if anchor:
        tf.vertical_anchor = anchor
    tf.margin_left = 0
    tf.margin_right = 0
    tf.margin_top = 0
    tf.margin_bottom = 0
    lines = s.split("\n")
    for i, ln in enumerate(lines):
        p = tf.paragraphs[0] if i == 0 else tf.add_paragraph()
        p.alignment = align
        if line_spacing:
            p.line_spacing = line_spacing
        r = p.add_run()
        r.text = ln
        r.font.name = font
        r.font.size = Pt(size)
        r.font.bold = bold
        r.font.italic = italic
        r.font.color.rgb = colour
    return box


def chrome(slide, eyebrow, header_right=COMPANY):
    bg(slide, WHITE)
    rect(slide, Inches(0), Inches(0), Inches(0.1), H, AMBER_D)
    rect(slide, Inches(0), Inches(0), W, Inches(1.0), BLACK)
    text(slide, eyebrow, Inches(0.3), Inches(0.38), Inches(7), Inches(0.35),
         size=12, bold=True, colour=OFF_WHITE, font="Calibri")
    text(slide, header_right, Inches(8.5), Inches(0.3), Inches(4.53), Inches(0.45),
         size=15, bold=True, colour=TEAL, align=PP_ALIGN.RIGHT, font="Georgia")
    text(slide, f"Prepared by Nitesh Roopa CA, Managing Partner and Founder, ProfitPulse, Profit-Pulse.com.au",
         Inches(0.3), Inches(7.05), Inches(9.5), Inches(0.3), size=8,
         colour=RGBColor(0x66, 0x66, 0x66), font="Calibri")
    text(slide, DATE_STR, Inches(11.5), Inches(7.05), Inches(1.53), Inches(0.3),
         size=8, colour=RGBColor(0x66, 0x66, 0x66), align=PP_ALIGN.RIGHT, font="Calibri")


# ============================================================
# SLIDE 1: COMMERCIAL INTELLIGENCE BRIEF
# ============================================================
s1 = prs.slides.add_slide(blank)
chrome(s1, "COMMERCIAL INTELLIGENCE BRIEF")

text(s1, "Orbitkey", Inches(0.3), Inches(1.15), Inches(9.5), Inches(0.7),
     size=40, bold=True, colour=BLACK, font="Georgia")
text(s1, "Design led everyday carry accessories brand, Richmond, Melbourne VIC",
     Inches(0.3), Inches(2.0), Inches(11.5), Inches(0.4), size=14,
     colour=RGBColor(0x33, 0x33, 0x33), font="Calibri")

# Stat cards, 5 across
labels = [
    ("$24.6M", "Annual revenue", "Smart50 2025 award"),
    ("44", "Team members", "Smart50 2025 award"),
    ("40%", "3 year revenue growth", "Export Awards 2025"),
    ("100+", "Countries exported to", "Export Awards 2025"),
    ("2013", "Founded", "SmartCompany history"),
]
card_w = Inches(2.3)
gap = Inches(0.15)
top = Inches(2.4)
card_h = Inches(1.6)
x = Inches(0.3)
for num, lbl, src in labels:
    rect(s1, x, top, card_w, card_h, BLACK)
    rect(s1, x, top, card_w, Pt(4), TEAL)
    text(s1, num, x + Inches(0.15), top + Inches(0.18), card_w - Inches(0.3), Inches(0.5),
         size=28, bold=True, colour=AMBER_B, font="Georgia")
    text(s1, lbl, x + Inches(0.15), top + Inches(0.78), card_w - Inches(0.3), Inches(0.5),
         size=11, colour=WHITE, font="Calibri", line_spacing=1.0)
    text(s1, src, x + Inches(0.15), top + Inches(1.34), card_w - Inches(0.3), Inches(0.22),
         size=8, colour=RGBColor(0xAA, 0xAA, 0xAA), italic=True, font="Calibri")
    x += card_w + gap

# Key commercial signals
text(s1, "KEY COMMERCIAL SIGNALS", Inches(0.3), Inches(4.3), Inches(6), Inches(0.3),
     size=12, bold=True, colour=TEAL, font="Calibri")

signals = [
    "Ranked 50th nationally, 2025 Smart50 Awards, fastest growing small businesses (SmartCompany).",
    "Winner, Creative Industries, Australian Export Awards 2025, Victorian state final (Export Awards).",
    "Ships from five third party logistics warehouses to over 100 export countries (Export Awards 2025).",
    "Raised over USD 3.25 million across six Kickstarter campaigns from 47,068 backers (SmartCompany).",
    "Registered as Orbitkey Pty Ltd, ABN 15 634 506 059, Richmond VIC (ASIC public register).",
]
y = Inches(4.68)
for sig in signals:
    rect(s1, Inches(0.3), y + Inches(0.08), Inches(0.08), Inches(0.08), AMBER_D)
    text(s1, sig, Inches(0.52), y, Inches(12.3), Inches(0.32), size=11,
         colour=RGBColor(0x22, 0x22, 0x22), font="Calibri")
    y += Inches(0.36)

# ============================================================
# SLIDE 2: THE OPPORTUNITY
# ============================================================
s2 = prs.slides.add_slide(blank)
chrome(s2, "THE OPPORTUNITY")
text(s2, "Orbitkey. Three commercial observations from ProfitPulse.",
     Inches(0.3), Inches(1.15), Inches(11.5), Inches(0.4), size=16, bold=True,
     colour=BLACK, font="Georgia")

col_w = Inches(4.15)
col_gap = Inches(0.08)
col_top = Inches(1.75)
col_h = Inches(5.0)
cols = [
    (TEAL, WHITE, "01", "Serial crowdfunding signals a cash gap",
     "Orbitkey's own founders say Kickstarter helps fund new products and manage cash "
     "flow around production. That is a direct signal that cash tied up in inventory, "
     "debtors and supplier terms across five global warehouses deserves a closer look.\n\n"
     "Working Capital Unlock maps exactly this: cash trapped in stock, work in progress "
     "and terms, with a prioritised release plan. Comparable clients typically release "
     "8 to 15 percent of revenue in freed up cash."),
    (BLACK, OFF_WHITE, "02", "Three product lines, one margin picture",
     "Orbitkey sells across Everyday Carry, Travel and Bag Organisation, and Work "
     "Organisation, through direct to consumer, Kickstarter and wholesale to over a "
     "thousand European stores. Three channels and three product families rarely carry "
     "identical margin.\n\n"
     "A line by line profitability view would show which products and channels are "
     "genuinely funding the growth, and which are being carried by the others."),
    (GOLD, BLACK, "03", "Fast international growth needs a costed plan",
     "Revenue has grown 40 percent over three years and Orbitkey now exports to more "
     "than 100 countries from five logistics warehouses. That pace of expansion usually "
     "outruns the financial plan supporting it.\n\n"
     "A Strategic Growth Diagnostic would map revenue, capacity and margin headroom "
     "into a costed 12 month plan, so the next stage of growth is funded on purpose."),
]
x = Inches(0.3)
for fill, txtcol, idx, head, body in cols:
    rect(s2, x, col_top, col_w, col_h, fill)
    text(s2, idx, x + Inches(0.25), col_top + Inches(0.2), col_w - Inches(0.5), Inches(0.6),
         size=28, bold=True, colour=txtcol, font="Georgia")
    text(s2, head, x + Inches(0.25), col_top + Inches(0.85), col_w - Inches(0.5), Inches(0.75),
         size=14, bold=True, colour=txtcol, font="Calibri", line_spacing=1.05)
    text(s2, body, x + Inches(0.25), col_top + Inches(1.65), col_w - Inches(0.5), Inches(3.1),
         size=10.5, colour=txtcol, font="Calibri", line_spacing=1.1)
    x += col_w + col_gap

text(s2, "These are observations offered in good faith. Orbitkey has built something "
     "genuinely impressive. The question is simply whether the financial architecture "
     "matches the pace of the growth.",
     Inches(0.3), Inches(6.85), Inches(12.7), Inches(0.35), size=10, italic=True,
     colour=RGBColor(0x44, 0x44, 0x44), font="Calibri")

# ============================================================
# SLIDE 3: THE RECOMMENDATION
# ============================================================
s3 = prs.slides.add_slide(blank)
chrome(s3, "THE RECOMMENDATION")

left_w = Inches(7.6)
text(s3, "Working Capital Unlock", Inches(0.3), Inches(1.2), left_w, Inches(0.5),
     size=24, bold=True, colour=BLACK, font="Georgia")
text(s3, "$6,000 one off", Inches(0.3), Inches(1.75), left_w, Inches(0.35),
     size=15, bold=True, colour=AMBER_D, font="Calibri")
text(s3, "A four week project mapping cash trapped in debtors, inventory, work in "
     "progress, supplier terms and bank facilities across Orbitkey's global operation, "
     "with a prioritised action list to release cash back into the business.",
     Inches(0.3), Inches(2.2), left_w, Inches(0.95), size=11.5,
     colour=RGBColor(0x22, 0x22, 0x22), font="Calibri", line_spacing=1.15)

rect(s3, Inches(0.3), Inches(3.3), left_w, Inches(1.55), RGBColor(0xF2, 0xF2, 0xF0),
     line=AMBER_D)
text(s3, "Step one, answer a few quick questions", Inches(0.5), Inches(3.45), left_w - Inches(0.4),
     Inches(0.35), size=13, bold=True, colour=BLACK, font="Calibri")
text(s3, "See the solutions matched to your size and industry.",
     Inches(0.5), Inches(3.82), left_w - Inches(0.4), Inches(0.3), size=11,
     colour=RGBColor(0x33, 0x33, 0x33), font="Calibri")
qtag = text(s3, "profit-pulse.com.au/services/find-your-fit", Inches(0.5), Inches(4.2),
     left_w - Inches(0.4), Inches(0.35), size=12, bold=True, colour=TEAL, font="Calibri")
qtag.text_frame.paragraphs[0].runs[0].hyperlink.address = "https://profit-pulse.com.au/services/find-your-fit/"

cta = rect(s3, Inches(0.3), Inches(5.05), left_w, Inches(0.55), TEAL)
ctatxt = text(s3, "Purchase the suggested product now to get started", Inches(0.3), Inches(5.2),
     left_w, Inches(0.3), size=12, bold=True, colour=WHITE, align=PP_ALIGN.CENTER, font="Calibri")
ctatxt.text_frame.paragraphs[0].runs[0].hyperlink.address = "https://buy.stripe.com/00w4gA7OUfGM5TadMH3ks0z"

text(s3, "Prefer a conversation first? Book a complimentary discovery call.",
     Inches(0.3), Inches(5.85), left_w, Inches(0.3), size=10.5,
     colour=RGBColor(0x33, 0x33, 0x33), font="Calibri")
booktag = text(s3, "Book a complimentary discovery call", Inches(0.3), Inches(6.15), left_w, Inches(0.3),
     size=10.5, bold=True, colour=TEAL, font="Calibri")
booktag.text_frame.paragraphs[0].runs[0].hyperlink.address = (
    "https://bookings.cloud.microsoft/book/ProfitPulse1@profit-pulse.com.au/?ismsaljsauthenabled=true")

# Right panel, About ProfitPulse
rx = Inches(8.1)
rw = Inches(4.93)
rect(s3, rx, Inches(1.2), rw, Inches(5.55), BLACK)
text(s3, "Nitesh Roopa", rx + Inches(0.25), Inches(1.4), rw - Inches(0.5), Inches(0.45),
     size=17, bold=True, colour=AMBER_B, font="Georgia")
text(s3, "CA, Managing Partner, ProfitPulse", rx + Inches(0.25), Inches(1.85), rw - Inches(0.5),
     Inches(0.3), size=11.5, colour=OFF_WHITE, font="Calibri")

pts = [
    "16 years across 4 countries",
    "52 deals executed and managed",
    "Largest single deal USD 1.3 billion, Cahora Bassa",
    "Total GRBT project value over AUD 10 billion",
]
y = Inches(2.35)
for p in pts:
    rect(s3, rx + Inches(0.25), y + Inches(0.07), Inches(0.07), Inches(0.07), TEAL)
    text(s3, p, rx + Inches(0.45), y, rw - Inches(0.7), Inches(0.4), size=10.5,
         colour=WHITE, font="Calibri", line_spacing=1.05)
    y += Inches(0.5)

rect(s3, rx + Inches(0.25), Inches(4.55), rw - Inches(0.5), Pt(1.5), TEAL)
contact = [
    "Profit-Pulse.com.au",
    "Nitesh@Profit-Pulse.com.au",
    "+61 411 876 267",
    "linkedin.com/in/nitesh-roopa-77594163",
]
y = Inches(4.75)
for c in contact:
    text(s3, c, rx + Inches(0.25), y, rw - Inches(0.5), Inches(0.35), size=11,
         colour=OFF_WHITE, font="Calibri")
    y += Inches(0.4)

out_path = "/home/user/Daily-Outreach/Out-reach efforts/Brief_Orbitkey_05Aug2026.pptx"
prs.save(out_path)
print(f"PPTX saved: {out_path}")
